chiselFrontend/src/main/scala/chisel3/Bits.scala

```diff
@@ -1830,216 +1807,193 @@ package experimental {
     }
 
     override def do_<< (that: Int)(implicit sourceInfo: SourceInfo, compileOptions: CompileOptions): Interval =
-      binop(sourceInfo, Interval(this.width + that, this.range << that), ShiftLeftOp, that)
+      binop(sourceInfo, Interval(this.range << that), ShiftLeftOp, that)
+
     override def do_<< (that: BigInt)(implicit sourceInfo: SourceInfo, compileOptions: CompileOptions): Interval =
       do_<<(that.toInt)
+
     override def do_<< (that: UInt)(implicit sourceInfo: SourceInfo, compileOptions: CompileOptions): Interval = {
-      val newRange = that.width match {
-        // TODO: (chick) should be this.range << (2^(w) - 1); leave firrtl to figure this out for now
-        //case w: KnownWidth => this.range << w
-        case _ => IntervalRange(firrtlir.UnknownBound, firrtlir.UnknownBound, this.range.binaryPoint)
-      }
-      binop(sourceInfo,
-        Interval(this.width.dynamicShiftLeft(that.width), newRange),
-        DynamicShiftLeftOp, that)
+      val newRange = IntervalRange(firrtlir.UnknownBound, firrtlir.UnknownBound, this.range.binaryPoint)
```

Comment:
Can you do the calculation now in Chisel if the UInt width is known, instead of returning an unknown range?